### PR TITLE
Fixed: prevented updating negative inventory counts (#207)

### PR DESCRIPTION
**Pull request by shanmukhdutt**
#207

Closes #

### Contribution and Currently Important Rules Acceptance

- [ ] I have read and followed [contribution rules](https://github.com/hotwax/inventory-count#contribution-guideline)

### CODE REVIEW

File: src/views/count.vue [hunk 1]
```diff
@@ -228,7 +228,9 @@
           }
         }])
         this.router.push('/search')
-        } else {
+        } else if(this.quantity < 0) {
+          showToast(translate("Negative stock count cannot be accepted"))
+        }else{
```
Comment (k2maan) [inline src/views/count.vue:233]: Improve spacing here

File: src/locales/en.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
   "Logging out": "Logging out",
   "Logout": "Logout",
   "Make sure you've reviewed the products and their counts before uploading them for review": "Make sure you've reviewed the products and their counts before uploading them for review",
+  "Neagative stock count cannot be accepted":"Neagative stock count cannot be accepted",
```
Comment (k2maan) [inline src/locales/en.json:34]: Please recheck the spellings

**Review by k2maan — CHANGES_REQUESTED** [review]

File: src/views/count.vue [hunk 3]
```diff
@@ -216,7 +216,7 @@
         await picker.present();
       },
       updateProductInventoryCount() {
-        if (this.quantity) {
+        if (this.quantity > 0) {
```
Comment (amansinghbais) [inline src/views/count.vue:219]: if(this.quantity > 0) {

Comment (amansinghbais) [inline src/views/count.vue:219]:
```suggestion
        if(this.quantity > 0) {
```

File: src/locales/en.json [hunk 4]
```diff
@@ -31,6 +31,7 @@
   "Logging out": "Logging out",
   "Logout": "Logout",
   "Make sure you've reviewed the products and their counts before uploading them for review": "Make sure you've reviewed the products and their counts before uploading them for review",
+  "Negative stock count cannot be accepted":"Negative stock count cannot be accepted",
```
Comment (amansinghbais) [inline src/locales/en.json:34]:
```suggestion
  "Negative stock count cannot be accepted": "Negative stock count cannot be accepted",
```

File: src/views/count.vue [hunk 5]
```diff
@@ -216,7 +216,7 @@
         await picker.present();
       },
       updateProductInventoryCount() {
-        if (this.quantity) {
+        if(this.quantity) {
```
Comment (ymaheshwari1) [inline src/views/count.vue:219]: We have not handled negative check in this case, and also we have added static text entry in locale but have added the toast anywhere

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Comment by ymaheshwari1** [comment]
Closing as having conflicts and needs improvements.

**Pull request closed without merging**